### PR TITLE
SQL functions for the rest of FormAccessors

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph 

:blowfish: 

cc @calellowitz @biyeun

### CODE REVIEW

**Comment by snopoke** [comment]
Sorry, history is a little messy in places

**Comment by benrudolph** [comment]
nice! these sql functions are really coming along. good stuff. waiting on tests, other g2g

**Comment by snopoke** [comment]
Damn, tests failing due to postgres incompatible. Travis uses 9.1 by default. I'll update the functions.

**Comment by benrudolph** [comment]
ouch

**Comment by benrudolph** [comment]
may be worth it to have travis just use 9.4. we're going to be upgrading soon anyways. https://github.com/travis-ci/travis-ci/issues/2983

**Comment by benrudolph** [comment]
(notably: https://github.com/travis-ci/travis-ci/issues/2983#issuecomment-75420000)

**Comment by snopoke** [comment]
Yes but can't do that until we have actually upgraded otherwise we won't be
able to deploy

File: corehq/form_processor/tests/test_form_dbaccessor.py [hunk 1]
```diff
@@ -1,28 +1,297 @@
...
+        self.assertEqual(1, len(transactions))
+        self.assertFalse(transactions[0].revoked)
+
+    def _check_simple_form(self, form):
         self.assertIsInstance(form, XFormInstanceSQL)
         self.assertIsNotNone(form)
         self.assertEqual(DOMAIN, form.domain)
         self.assertEqual('user1', form.user_id)
+        return form
+
```
Comment (dimagimon) [inline corehq/form_processor/tests/test_form_dbaccessor.py]: W293 blank line contains whitespace

File: corehq/form_processor/tests/test_form_dbaccessor.py [hunk 2]
```diff
@@ -1,28 +1,297 @@
...
+    form_data = _get_form_data(form_id, case_id)
+
+    form = XFormInstanceSQL(
+        form_id=form_id,
+        xmlns='http://openrosa.org/formdesigner/form-processor',
+        received_on=utcnow,
+        user_id=user_id,
+        domain=DOMAIN
+    )
+
```
Comment (dimagimon) [inline corehq/form_processor/tests/test_form_dbaccessor.py]: W293 blank line contains whitespace

**Comment by snopoke** [comment]
:white_check_mark: finally

**Comment by snopoke** [comment]
bump

**Comment by benrudolph** [comment]
looks good to me. thanks for the explanations

**Pull request merged**